docs/2.10.0/docs/canton/usermanual/upgrading.rst

```diff
@@ -211,6 +215,60 @@ The ping command creates two contracts between the admin parties, then exercises
 Version Specific Notes
 ~~~~~~~~~~~~~~~~~~~~~~
 
+.. _upgrade_to_2.10:
+
+Upgrade to Release 2.10
+^^^^^^^^^^^^^^^^^^^^^^^
+
+Protocol Versions
+"""""""""""""""""
+2.10 is a Long Term Support (LTS) release and introduces a new Canton protocol version 7, which is the
```

Comment:
I am trying to clarify if we want to call 2.10 an LTS already.

Reply:
I think this is a good point and we don't want to do it right now.

Reply:
yes we don't call it LTS yet, we will do that later once we have confirmed it is stable enough.